packages/hooks/src/useP2PAdvertList.ts

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
-import { usePaginatedFetch } from '@deriv/api';
+import { useQuery } from '@deriv/api';
 import useExchangeRate from './useExchangeRate';
 
 /**
  * This custom hook returns available adverts for use with 'p2p_order_create' by calling 'p2p_advert_list' endpoint
  */
-const useP2PAdvertList = (
-    payload?: NonNullable<Parameters<typeof usePaginatedFetch<'p2p_advert_list'>>[1]>['payload']
-) => {
+const useP2PAdvertList = (payload?: NonNullable<Parameters<typeof useQuery<'p2p_advert_list'>>[1]>['payload']) => {
```

Comment:
Can we de-structure these payloads?

Reply:
@niloofar-deriv,  I think not `destructuring` the `payload` `object` in this case is better, because as seen in the screenshot below, we `spread` it directly into the `payload` `object` of `useQuery`. `Destructuring` it here would mean that every time someone adds a new `property` to the `payload`, they'd have to modify this file as well. 

I believe we should not `destructure` it, but if you think it's better that way, The I can effect the changes

<img width="944" alt="payload" src="https://github.com/binary-com/deriv-app/assets/104334373/85d14445-8ab1-4d4e-8454-0cc6cad96454">

Reply:
Oh I see, I didn't notice that part
You can ignore my comment 🙏